tests/messages/test_messages.py

```diff
@@ -117,5 +117,5 @@ def test_from_hex_sysex_data_type():
 
 def test_repr():
     msg = Message('note_on', channel=1, note=2, time=3)
-    msg_eval = eval(repr(msg))
+    msg_eval = eval(repr(msg))  # noqa: S307
```

Comment:
I believe using the recommended `ast.literal_eval()` should work here to mitigate the issue.

Reply:
Oh, my bad!
It's in the name: `literal`, duh…
All good then :)